source/Energinet.DataHub.MeteringPoints.Infrastructure/Integration/IntegrationEvents/CreateMeteringPoint/MeteringPointCreatedEventMessageMapper.cs

```diff
@@ -31,9 +31,9 @@ protected override IMessage Convert(MeteringPointCreatedEventMessage obj)
                 MeteringPointId = obj.MeteringPointId,
                 GsrnNumber = obj.GsrnNumber,
                 MeteringPointType = obj.GetMeteringPointType(),
-                GridAreaCode = obj.GridAreaId,
-                ToGridAreaCode = obj.ToGrid,
-                FromGridAreaCode = obj.FromGrid,
+                GridAreaLink = obj.GridAreaId,
+                ToGridAreaLink = obj.ToGrid,
+                FromGridAreaLink = obj.FromGrid,
```

Comment:
From this code block, I can infer that `grid area link = grid area id = grid`.
That doesn't look correct to me. But I don't know whether it's code errors or bad inconsistent naming.

Reply:
I'd probably expect to see something similar to

```suggestion
                GridAreaLinkId = obj.GridAreaLinkId,
                ToGridAreaLinkId = obj.ToGridAreaLinkId,
                FromGridAreaLinkId = obj.FromGridAreaLinkId,
```

Reply:
Inconsistent naming.

Reply:
Fixed